requirements.txt

```diff
@@ -1,14 +1,14 @@
 absl-py
 packaging
 pandas
-tensorflow
+tf-nightly-cpu==2.16.0.dev20231103  # Pin a working nightly until rc0.
```

Comment:
Also, we should install `tf-nightly` without the `-cpu`.  If the user's machine has GPU, this might lead to installing a CPU only version.  For GitHub workflow, `tf-nightly` would be suffice.

Reply:
done